spyderlib/widgets/tabs.py

```diff
@@ -112,7 +135,8 @@ def __init__(self, parent, actions=None, menu=None,
         QTabWidget.__init__(self, parent)
         
         self.setUsesScrollButtons(True)
-        
+        self.filter = TabFilter(parent)  # Need to keep reference

```

Comment:
I don't like this too much because the filter is installed for all tabs, but only works for tabs on editor_stack's(?)

Does this have a benefit for our other tabbed widgets or not?